src/util/fields.py

```diff
@@ -372,6 +376,11 @@ def adapter(leaf, is_or, negated):
         remove_field(cr, inh.model, fieldname, cascade=cascade, drop_column=drop_column, skip_inherit=skip_inherit)
 
 
+def unstore_field(cr, model, field):
```

Comment:
I'd suggest a name more in line with what we already have: 
```suggestion
def make_field_non_stored(cr, model, field):
```